Introduction.md: Fix FIXME in "Garbage collection in WebKit"

tetsuharuohzeki opened the pull request:
#### 064084f23e049f96589bf424c9b331bb6f4bb725
<pre>
Introduction.md: Fix FIXME in &quot;Garbage collection in WebKit&quot;
<a href="https://bugs.webkit.org/show_bug.cgi?id=244458">https://bugs.webkit.org/show_bug.cgi?id=244458</a>

Reviewed by Ryosuke Niwa.

* Introduction.md:

Canonical link: <a href="https://commits.webkit.org/253893@main">https://commits.webkit.org/253893@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/864f2f7cf88bca011d9cd205e08c46c6ca96ddf7

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/87409 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/31497 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/18217 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/96639 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/29862 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/26052 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/79527 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/91410 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/93027 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/68/builds/24126 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/74198 "Built successfully") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/23965 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/79096 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/79322 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/66982 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/27576 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/13154 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/27528 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/14170 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/29216 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/37032 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1100 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/29145 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/33448 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

Constellation commented on Introduction.md:
```diff
@@ -468,7 +468,11 @@ WebKit uses two primary management strategies when objects in other cases:
+See these blog posts:
+
+* [Understanding Garbage Collection in JavaScriptCore From Scratch](https://webkit.org/blog/12967/understanding-gc-in-jsc-from-scratch/)
+* [Introducing Riptide: WebKit’s Retreating Wavefront Concurrent Garbage Collector](https://webkit.org/blog/7122/introducing-riptide-webkits-retreating-wavefront-concurrent-garbage-collector/)
```
I think it is helpful if we describe the brief summary of each blog post here, for example,

> Understanding Garbage Collection in JavaScriptCore From Scratch: Summary of how our GC is organized and how it works.
> Introducing Riptide: WebKit’s Retreating Wavefront Concurrent Garbage Collector: Focusing on novel algorithms of our GC including how to tackle web browser specific problems (managing DOM objects, concurrency, etc.).

tetsuharuohzeki replied:
Make sense. I addressed it. https://github.com/WebKit/WebKit/pull/3754/commits/864f2f7cf88bca011d9cd205e08c46c6ca96ddf7

rniwa reviewed (approved):
r=me

tetsuharuohzeki committed:
Introduction.md: Fix FIXME in "Garbage collection in WebKit"
https://bugs.webkit.org/show_bug.cgi?id=244458

Reviewed by Ryosuke Niwa.

* Introduction.md:

Canonical link: https://commits.webkit.org/253893@main
